js/ui/handler/box_zoom.js

```diff
@@ -21,6 +23,9 @@ function BoxZoomHandler(map) {
 
 BoxZoomHandler.prototype = {
 
+    enabled: false,
+    active: false,
```

Comment:
Some of the handlers, such as  <code>BoxZoomHandler</code>, already had a public (no underscore), albeit non-documented <code>active</code> property on them, I just took the opportunity to make code and behaviour consistent with the new enabled property.